src/amo/reducers/home.js

```diff
@@ -314,7 +329,11 @@ export const createInternalHeroShelves = (
   }
 
   if (primary === null) {
```

Comment:
Same here, when would `primary` be `null`?

Reply:
This was a part of the original `createInternalHeroShelves` that was updated to handle cases in which there was missing hero data.